toolchain/lowering/lowering_context.cpp

```diff
@@ -72,13 +71,30 @@ auto LoweringContext::BuildFunctionDeclaration(SemIR::FunctionId function_id)
   // TODO: Lower type information for the arguments prior to building args.
   auto param_refs = semantics_ir().GetNodeBlock(function.param_refs_id);
   llvm::SmallVector<llvm::Type*> args;
-  args.resize_for_overwrite(first_param + param_refs.size());
+  llvm::SmallVector<SemIR::NodeId> param_nodes;
```

Comment:
It might be worth calling this `param_node_ids` to emphasize that it's an id rather than a node.

Reply:
Done.